File: C/LeetCode/9.c
#include <stdio.h>
#include <stdbool.h>

bool isPalindrome(int x) 
{
    if (x < 0)
    {
        return false;
    }
    long a = x, b = 0;
    while (x > 0)
    {
        long c = x % 10;
        b = b * 10 + c;
        x /= 10;
    }
    if(a == b)
    {
        return true;
    }
    else
    {
        return false;
    }
    

}

File: Python/LeetCode/9.py
class Solution(object):
    def isPalindrome(self, x):
        a = x
        b = 0
        if(x < 0):
            return False
        while(x > 0):
            c = x % 10
            b = b * 10 + c
            x /= 10

        if(a == b):
            return True
        else:
            return False
